modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/AttachmentObjectFieldBusinessType.java

```diff
@@ -94,6 +96,15 @@ public Map<String, Object> getProperties(
 			ObjectFieldSettingConstants.NAME_STORAGE_DL_FOLDER_PATH);
 
 		return HashMapBuilder.<String, Object>put(
+			"objectDefinitionScope",
+			() -> {
+				ObjectDefinition objectDefinition =
+					_objectDefinitionLocalService.getObjectDefinition(
+						objectField.getObjectDefinitionId());
+
+				return objectDefinition.getScope();
+			}
```

Comment:
@seltong can we change to use the `objectScopeProvider.isGroupAware()`? Renaming the property to `groupAware`.

Thanks!

Reply:
Yeah, we can use it!

Thanks!